notebook/converted/benchmark_results.py

```diff
@@ -1158,9 +1160,54 @@ def return_unlabeled():
 tissue_metrics
 
 
+# In[7]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += geom_hline(yintercept=.5, linetype='dashed')
+plot += ggtitle('Blood vs Breast Tissue Prediction')
+plot
+
+
+# ### All Tissue Predictions
+
+# In[8]:
+
+
+in_files = glob.glob('../../results/all-tissue.*.tsv')
+in_files = [f for f in in_files if 'be_corrected' not in f]
+print(in_files[:5])
+
+
 # In[9]:
 
 
+tissue_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('all-tissue.')[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+        
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('deep_net', 'five_layer_net')
+tissue_metrics
+
+
+# In[10]:
+
+
 plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
```

Comment:
It's definitely interesting that the three-layer and seven-layer networks perform so much worse than the five-layer net. Maybe the learning rate or other hyperparameters could be affecting this? I guess it makes sense to avoid deeper networks if you're fairly sure it won't be an easy fix.

Reply:
Yeah for sure. I think it's some combination of being a good depth for the problem space and not running into vanishing gradients.